plasma_framework/contracts/src/exits/payment/controllers/PaymentStartStandardExit.sol

```diff
@@ -99,28 +100,23 @@ library PaymentStartStandardExit {
         bool isTxDeposit = controller.isDeposit.test(utxoPos.blockNum());
         uint192 exitId = ExitId.getStandardExitId(isTxDeposit, args.rlpOutputTx, utxoPos);
         (bytes32 root, uint256 blockTimestamp) = controller.framework.blocks(utxoPos.blockNum());
-        bytes memory outputGuardData = args.outputType == 0 ? bytes("") : args.outputGuardData;
-
-        address payable exitTarget;
-        IOutputGuardParser outputGuardParser;
-        if (args.outputType == 0) {
```

Comment:
This is the main cleanup, no longer need to rely on a preserved "output type 0".